subprojects/gradle/test-summary/src/main/kotlin/com/avito/test/summary/model/CrossDeviceSuite.kt

```diff
@@ -25,25 +26,34 @@ public data class CrossDeviceSuite(val crossDeviceRuns: List<CrossDeviceRunTest>
     val lostOnSomeDevicesCount: Int
         get() = consistentData.count { it.status is CrossDeviceStatus.LostOnSomeDevices }
 
-    val percentLostOnSomeDevicesOfAutomated: Percent
-        get() = lostOnSomeDevicesCount.percentOf(automatedCount)
-
-    val inconsistentCount: Int
-        get() = crossDeviceRuns.count { it.status is CrossDeviceStatus.Inconsistent }
-
-    val percentSuccessOfAutomated: Percent = success.percentOf(automatedCount)
-
-    val percentSkippedOnAllDevicesOfAutomated: Percent = skippedOnAllDevicesCount.percentOf(automatedCount)
-
     val failedOnAllDevicesCount: Int
         get() = consistentData.count { it.status is CrossDeviceStatus.FailedOnAllDevices }
 
-    val percentFailedOnAllDevicesOfAutomated: Percent = failedOnAllDevicesCount.percentOf(automatedCount)
-
     val failedOnSomeDevicesCount: Int
         get() = consistentData.count { it.status is CrossDeviceStatus.FailedOnSomeDevices }
 
-    val percentFailedOnSomeDevicesOfAutomated: Percent = failedOnSomeDevicesCount.percentOf(automatedCount)
+    val percentLostOnSomeDevicesOfAutomated: Percent
+    val percentSuccessOfAutomated: Percent
+    val percentSkippedOnAllDevicesOfAutomated: Percent
+    val percentFailedOnAllDevicesOfAutomated: Percent
+    val percentFailedOnSomeDevicesOfAutomated: Percent
```

Comment:
It looks like the Hungarian notation. I think it's excessive.

Reply:
Agree